glossary.yml

```diff
@@ -5332,23 +5422,23 @@
   en:
     term: "truthy"
     def: >
-      Evaluating to true in a [Boolean](#boolean) context.
+      Evaluating to [true](#true) in a [Boolean](#boolean) context.
 
 
 - slug: tuple
   en:
     term: "tuple"
     def: >
-      A value that has multiple parts, such as the three color components of a
-      red-green-blue color specification.
+      A data type that has a fixed number of parts, such as the three color components of a
+      red-green-blue color specification. Tuples are immutable (their values can not be reset.)
 
 
 - slug: two_hard_problems
   en:
     term: "two hard problems in computer science"
     def: >
-      Refers to a quote by Phil Karlton: "There are only two hard problems in computer
-      science: cache invalidation and naming things." Many variations add a third
+      Refers to a quote by Phil Karlton–"There are only two hard problems in computer
+      science–cache invalidation and naming things." Many variations add a third
```

Comment:
Needs to be an em-dash, not en-dash

```suggestion
      science—cache invalidation and naming things." Many variations add a third
```

Reply:
That's what I tried to type, but it wasn't working—my keyboard + vim didn't want to do them. But it looked like the colons were wreaking havoc with the syntax highlighting, and I thought that might pose other problems.